src/Microsoft.IdentityModel.Validators/AadTokenValidationParametersExtension.cs

```diff
@@ -101,6 +115,26 @@ internal static bool ValidateIssuerSigningKey(SecurityKey securityKey, SecurityT
             return true;
         }
 
+        private static string GetTid(SecurityToken securityToken)
+        {
+            if (securityToken is JsonWebToken jsonWebToken)
+            {
+                if (jsonWebToken.TryGetPayloadValue<string>("tid", out string tid))
+                    return tid;
+
+                return string.Empty;
+            }
+            else if (securityToken is JwtSecurityToken jwtSecurityToken)
```

Comment:
Unnecessary `else` since previous block returns.
```suggestion
            if (securityToken is JwtSecurityToken jwtSecurityToken)
```